src/Forms/FixCommonErrors.cs

```diff
@@ -2654,20 +2633,19 @@ private void DialogsOnOneLine()
             for (int i = 0; i < Subtitle.Paragraphs.Count; i++)
             {
                 Paragraph p = Subtitle.Paragraphs[i];
-                string oldText = p.Text;
-                var text = FixCommonErrorsHelper.FixDialogsOnOneLine(oldText, language);

```

Comment:
I think since its going to be run even after FixCommonErrorsHelper.FixDialogsOnOneLine(..., ...) why not run it first to check if it should proceed or not